src/Tizen.NUI/src/internal/Interop/Interop.CanvasView.cs

```diff
@@ -0,0 +1,35 @@
+﻿/*
+ * Copyright(c) 2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+ 
+namespace Tizen.NUI
+{
+    internal static partial class Interop
+    {
+        internal static partial class CanvasView
+        {
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_CanvasView_New__SWIG_0")]
```

Comment:
@JSUYA 
Hello,
I think the dali patches that you have mentioned in description should be merged prior to this patch is reviewed.
otherwise there can be a runtime exception which blocks the release of tizenfx by TCT hurdle.

now I put the **NO MERGE** tag on thie PR.
you can remove **NO MERGE** tag for yourself when this PR is fixed and ready to be merged.
(**NO MERGE** tag has no restriction to submit PR or to editing this patch but just for notification and marking.
thank you.